docs/README.md

```diff
@@ -156,22 +225,23 @@ jobs:
   sync:
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout
+      - name: Checkout
```

Comment:
My editor re-formatted the YAML on save (using Prettier). I think this is OK, and the YAML looks a bit nicer like this, to me at least.

Reply:
I agree, I'd just copied GitHub's official docs which were odd. I prefer this.